crates/sargon/src/wrapped_radix_engine_toolkit/low_level/v1/transaction_manifest/transaction_manifest.rs

```diff
@@ -343,10 +458,16 @@ BURN_RESOURCE
     #[test]
     fn manifest_summary_simple() {
         let manifest = SUT::sample();
-        let summary = manifest.summary();
+        let summary = manifest.summary().unwrap();
         pretty_assertions::assert_eq!(
             summary,
             ManifestSummary::new(
+                hashmap!(
```

Comment:
oh where is this `hashmap!` macro? from RET? From Scrypto? 3rd party crate? or `std` ?

Reply:
From Scrypto. There's also `indexset!`, `indexmap!`